pootle/apps/pootle_store/forms.py

```diff
@@ -280,11 +284,16 @@ def clean_target_f(self):
 
             return value
 
-        def clean_state(self):
+        def _clean_state(self):
+            # Handle clean_state here because it depends on clean_suggestion.
```

Comment:
i meant more to move the logic into clean, because that is the method called *after* all the other validators have been called, so its expected.

with it named _clean_state its confusing that its actually called after